2022-02-04 02:51:42,814 INFO    MainThread:16360 [wandb_setup.py:_flush():71] setting env: {}
2022-02-04 02:51:42,814 INFO    MainThread:16360 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-04 02:51:42,815 INFO    MainThread:16360 [wandb_init.py:_log_setup():371] Logging user logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_025142-3b4lai8a\logs\debug.log
2022-02-04 02:51:42,815 INFO    MainThread:16360 [wandb_init.py:_log_setup():372] Logging internal logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_025142-3b4lai8a\logs\debug-internal.log
2022-02-04 02:51:42,815 INFO    MainThread:16360 [wandb_init.py:init():404] calling init triggers
2022-02-04 02:51:42,815 INFO    MainThread:16360 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-04 02:51:42,816 INFO    MainThread:16360 [wandb_init.py:init():460] starting backend
2022-02-04 02:51:42,816 INFO    MainThread:16360 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-02-04 02:51:42,830 INFO    MainThread:16360 [backend.py:ensure_launched():216] starting backend process...
2022-02-04 02:51:42,926 INFO    MainThread:16360 [backend.py:ensure_launched():222] started backend process with pid: 19556
2022-02-04 02:51:42,927 INFO    MainThread:16360 [wandb_init.py:init():469] backend started and connected
2022-02-04 02:51:42,928 INFO    MainThread:16360 [wandb_init.py:init():533] updated telemetry
2022-02-04 02:51:43,013 INFO    MainThread:16360 [wandb_init.py:init():563] communicating current version
2022-02-04 02:51:44,988 INFO    MainThread:16360 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-04 02:51:44,991 INFO    MainThread:16360 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-04 02:51:45,370 INFO    MainThread:16360 [wandb_init.py:init():606] starting run threads in backend
2022-02-04 02:51:48,207 INFO    MainThread:16360 [wandb_run.py:_console_start():1810] atexit reg
2022-02-04 02:51:48,218 INFO    MainThread:16360 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-04 02:51:48,218 INFO    MainThread:16360 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-04 02:51:48,220 INFO    MainThread:16360 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-04 02:51:48,220 INFO    MainThread:16360 [wandb_init.py:init():633] run started, returning control to user process
2022-02-04 02:51:48,981 INFO    MainThread:16360 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-04 02:51:48,982 INFO    MainThread:16360 [wandb_run.py:_restore():1752] restore
2022-02-04 02:51:49,787 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 02:51:49,904 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 02:51:50,261 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 02:51:50,477 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7410
}

2022-02-04 02:51:50,594 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7410
}

2022-02-04 02:51:50,703 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7410
}

2022-02-04 02:51:50,818 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7410
}

2022-02-04 02:51:50,925 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 7410
}

2022-02-04 02:51:51,033 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:51,145 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:51,257 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:51,368 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:51,479 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:52,272 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}

2022-02-04 02:51:53,085 INFO    MainThread:16360 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7410
  total_bytes: 7410
}
local_info {
}

2022-02-04 02:51:54,279 INFO    MainThread:16360 [wandb_run.py:_append_files():2180] logging synced files
